db/migration/1733228214376-CreateDonors.ts

```diff
@@ -0,0 +1,24 @@
+import { MigrationInterface, QueryRunner } from "typeorm"
+
+export class CreateDonors1733228214376 implements MigrationInterface {
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        await queryRunner.query(`-- sql
+            CREATE TABLE donors (
```

Comment:
This table will be read and written to from the scripts that we use to update the donor list. Previously the scripts generated HTML, now they will use this table. That part will be reviewed and QA'd by Lars.